src/components/Progress/Progress.js

```diff
@@ -10,7 +10,7 @@ const progressInfoFormatters = {
   messagedPercentage: (current, total, payload) => (
     <React.Fragment>
```

Comment:
Consider using named import for Fragment
```suggestion
    <Fragment>
          // ...
    </Fragment>
```

Reply:
Agree with that.